code/addons/interactions/template/stories/basics.stories.ts

```diff
@@ -46,23 +51,30 @@ export const Callback = {
 // an explicit test here
 export const SyncWaitFor = {
   play: async ({ canvasElement, step }) => {
-    const canvas = within(canvasElement);
+    const canvas = safeWithin(canvasElement);
     await step('Setup', Callback.play);
-    await waitFor(() => canvas.getByText('Completed!!'));
+
+    // FIXME: why doesn't this work?
```

Comment:
@tmeasday I had to change this to get web-components to work due to some shadowRoot-related issue that I don't understand. Is this ok?

Reply:
We took a look at this and (semi-) resolved it. It was due to bundling different versions (`.js` + `.mjs`) of the instrumenter somehow.